web/app/helpers/get-link-icon.ts

```diff
@@ -0,0 +1,54 @@
+import { helper } from "@ember/component/helper";
+import { assert } from "@ember/debug";
+
+export interface GetLinkIconSignature {
+  Args: {
+    Positional: [string | undefined];
+  };
+  Return: string;
+}
+
+const getLinkIconHelper = helper<GetLinkIconSignature>(([url]) => {
+  if (url) {
+    const urlParts = url.split("/");
+    let domain = urlParts[2];
```

Comment:
Should we have more URL validation at this level? What kind of UX is it if we fail the `assert` below?